Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openjdk:11.0.13-jdk-bullseye
```

Comment:
Blocked by https://github.com/CS4850Group5A/VirtuDoc/issues/43 and https://github.com/CS4850Group5A/VirtuDoc/pull/44 as better unit testing is needed on Docker images before allowing them through.